plugins/notifications-backend/src/service/handlers.ts

```diff
@@ -353,7 +353,9 @@ function createQuery(
   // filter by text
   if (filter.containsText) {
     query.andWhere(function () {
-      this.whereILike('title', `%${filter.containsText}%`).orWhereILike(
+      // case-insensitive "whereILike" should be used instead but
```

Comment:
it should be ilike. i would like to know what is the bug before considering a change in the code.

Reply:
```
Request failed with status 500 select * from `messages` left join (select * from `users` where `users`.`user` = 'default/guest') as `users` on `messages`.`id` = `users`.`message_id` where ((`is_system` = false and (`id` in (select `message_id` from `users` where `user` = 'default/guest')))) and (`title` ilike '%m%' or `message` ilike '%m%') and `created` > '2024-01-10T15:11:16.204Z' and (`read` = false or `read` is null) order by `created` desc limit 5 - near "ilike": syntax error type=errorHandler code=SQLITE_ERROR stack=SqliteError: select * from `messages` left join (select * from `users` where `users`.`user` = 'default/guest') as `users` on `messages`.`id` = `users`.`message_id` where ((`is_system` = false and (`id` in (select `message_id` from `users` where `user` = 'default/guest')))) and (`title` ilike '%m%' or `message` ilike '%m%') and `created` > '2024-01-10T15:11:16.204Z' and (`read` = false or `read` is null) order by `created` desc limit 5 - near "ilike": syntax error
```


Check the referenced knex issue, it's a bug and this is a workaround

Reply:
we must not change it for all DBs just because there's a bug in knex with sqlite. we can try other solutions

Reply:
I agree. Just we should find it today. Or merge this and keep searching.
Obvious workaround can be an `if` for the SQLite but maybe there is something better.